attest/core/src/types/report_body.rs

```diff
@@ -217,25 +224,20 @@ impl Ord for ReportBody {
     /// Create an arbitrary sort order for report body types
     ///
     /// We sort by Family ID, ProdID, Extended ProdID, SVN, MrSigner, MrEnclave,
-    /// Attributes, Misc Select, ConfigId, ConfigSVN, CPU SVN, and
-    /// ReportData, in that order
+    /// Misc Select, ConfigId, ConfigSVN, CPU SVN, and ReportData, in that order
     fn cmp(&self, other: &Self) -> Ordering {
         match self.0.isv_family_id[..].cmp(&other.0.isv_family_id[..]) {
             Ordering::Equal => match self.0.isv_prod_id.cmp(&other.0.isv_prod_id) {
                 Ordering::Equal => match self.0.isv_ext_prod_id.cmp(&other.0.isv_ext_prod_id) {
                     Ordering::Equal => match self.0.isv_svn.cmp(&other.0.isv_svn) {
                         Ordering::Equal => match self.mr_signer().cmp(&other.mr_signer()) {
                             Ordering::Equal => match self.mr_enclave().cmp(&other.mr_enclave()) {
-                                Ordering::Equal => match self.attributes().cmp(&other.attributes())
-                                {
-                                    Ordering::Equal => {
-                                        match self.0.misc_select.cmp(&other.0.misc_select) {
-                                            Ordering::Equal => {
-                                                match self.config_id().cmp(&other.config_id()) {
-                                                    Ordering::Equal => match self
-                                                        .0
-                                                        .config_svn
-                                                        .cmp(&other.0.config_svn)
+                                Ordering::Equal => {
```

Comment:
I added a task to https://github.com/mobilecoinfoundation/mobilecoin/issues/3437, in order to capture that Attributes is no longer considered and may need to be brought back in.

I think this isn't needed, but the macro that `ReportBody` uses requires it. Once we move over to `mc-sgx-core-types::ReportBody` I think it will be a non issue

Reply:
🤔 any idea what happens if you complete remove `Ord` on `ReportBody`?
What sorts them? I wonder what effect altering the behavior of this sort would have...

Reply:
you can see on #3449 where Ord is completely removed all the way up through `Report`.
It couldn't be done prior without completely swapping out `ReportBody` becuase the `SgxWrapper` macro requires `Ord`.

Reply:
Per discussion offline, we're not worried about `Ord` at this time. The higher type that wanted the `Ord` was unused and has been removed in #3454